docs/src/main/sphinx/release/release-355.md

```diff
@@ -0,0 +1,39 @@
+# Release 355 (XX Apr 2021)
+
+## General
+
+* Add support for configuring multiple password authentication plugins. ({issue}`7151`)
+* Rename `optimizer.plan-with-table-node-partitioning` config property to `optimizer.use-table-scan-node-partitioning`. ({issue}`7257`)
+* Report tables directly referenced by query to event listener. ({issue}`7330`)
+* Report columns which are created, inserted or updated to event listener. ({issue}`7425`)
+* Planning with table scan node partitioning is now triggered automatically
+  when ratio between table buckets and possible table scan tasks exceeds or is equal to
+  `optimizer.table-scan-node-partitioning-min-bucket-to-task-ratio`. This helps improve
+  query parallelism when table bucket count is small compared to number of nodes. ({issue}`7257`)
+* Include information about {doc}`/admin/spill` in {doc}`/sql/explain-analyze`. ({issue}`7427`)
```

Comment:
Is this needed?  If so, maybe "Improve documentation for spilling administration"

Reply:
it's about improving EXPLAIN ANALYZE ouytput, not its docs.